packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -503,7 +503,7 @@ export const ForumEventPoll = ({
   const voteClusters = useMemo(
     () => clusterForumEventVotes({ voters: votersRef.current, comments, event, currentUser }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [votersRef.current, event, currentUser]
+    [votersRef.current, event, currentUser, comments]
```

Comment:
Fixes a bug where if you start on a post page and navigate to the frontpage sometimes the comments haven't loaded